cmd/launcher/main.go

```diff
@@ -27,27 +27,28 @@ import (
 )
 
 func main() {
-	// create initial logger. As this is prior to options parsing,
-	// use the environment to determine verbosity.  It will be
-	// re-leveled during options parsing.
-	logger := logutil.NewServerLogger(env.Bool("LAUNCHER_DEBUG", false))
+	systemSlogger, logCloser, err := systemSlogger()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error creating system logger: %v\n", err)
+		os.Exit(1)
+	}
+	defer logCloser.Close()
```

Comment:
should we close this just before starting the windows svc? it have two windows event writers does not seem to be an issue 🤷

Reply:
yea, it would be an annoying refactor, also weird passing around a closer

Reply:
I'm sure we want the refactor. I'm less sure we want it _today_ (I'm not sure how hard it would be, it's just adding slogger as a parameter to the subcommands, right?)

Reply:
seems like we would also need to pass around the closer within main.go if we don't want to use defer .... I'm torn between wanting clean code and wanting to make sure we close the handler before we exit

Reply:
> I'm sure we want the refactor. I'm less sure we want it _today_ (I'm not sure how hard it would be, it's just adding slogger as a parameter to the subcommands, right?)

Yea pass slogger as a parameter and this event handler that needs to be closed, you could use an interface that has a close method on it.

Reply:
we could do something like

```go
func main() {
	if err := runMain(); err != nil {
		os.Exit(1)
	}
}

func runMain() error {
        // do stuff
        // return an err or nil
}
```

Reply:
Yeah, that's not bad. I was thinking of something much more complicated:
```
var exitDeferred []func

func osExit(ret int) {
  for _, fn := range exitDeferred {
    fn()
  }
  os.Exit(ret)
}

func main() {
  exitDeferred = append(exitDeferred, ...)
  osExit(0)
}

Reply:
I think both of these are good options 👍 since we already have 2 other defer funcs in here that we'd probably want to move around too, I'm fine w punting this refactor to a subsequent PR in order to merge this one

Reply:
Adding system multislogger to subcommands: https://github.com/kolide/launcher/pull/1674